keanu-project/src/main/java/io/improbable/keanu/algorithms/mcmc/NetworkSamplesGenerator.java

```diff
@@ -0,0 +1,72 @@
+package io.improbable.keanu.algorithms.mcmc;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import io.improbable.keanu.algorithms.NetworkSamples;
+import io.improbable.keanu.network.NetworkState;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Accessors(fluent = true)
```

Comment:
Do we definitely want to use "experimental" features from Lombok?

Reply:
The only risk is that they remove the feature or if the feature is fundamentally broken. Our test ensure it's not broken and it looks like they think it'll be moved to the "stable" category soon. `Current status: positive - Currently we feel this feature may move out of experimental status with no or minor changes soon.`.

Reply:
Yeah - I'd spotted that.  Just wanted to make sure we were making the decision with our eyes open though.  Thanks for thinking about it.